pandas/tests/frame/test_reshape.py

```diff
@@ -1128,3 +1128,34 @@ def test_stack_timezone_aware_values():
         ),
     )
     tm.assert_series_equal(result, expected)
+
+
+def test_unstacking_multi_index_df():
+    # BUG: gh-30740
```

Comment:
again just for consistency
```suggestion
    # see gh-30740
```